archetype-common/src/main/resources/maven-4.0.0.xsd

```diff
@@ -0,0 +1,2246 @@
+<?xml version="1.0"?>
```

Comment:
strange that this has to be copied into this project. I'd expect it to be loaded from maven core, if at all.

Reply:
Feel free to point me to location of the `xsd` in maven core or any other way to load it and I happily update the PR.